jannistsiroyannis/officewars
# Add initial docker support

This branch will:
- Add build support using docker-compose
- Add serve support using docker-compose
- Add additional error checks for file operations
- Remove build step from `run.sh`

This branch should not affect any of the previous functionality of the game

To test:
- Install docker and docker-compose
- `docker-compose run build`
- `docker-compose up serve`
- Go to: `localhost:8000`
- Make sure that the game behaves as expected
